specs/phase1/beacon-chain.md

```diff
@@ -727,7 +752,8 @@ def apply_shard_transition(state: BeaconState, shard: Shard, transition: ShardTr
     assert bls.AggregateVerify(zip(pubkeys, signing_roots), signature=transition.proposer_signature_aggregate)
 
     # Save updated state
-    state.shard_states[shard] = transition.shard_states[-1]
+    state.shard_states[shard] = transition.shard_states[len(transition.shard_states) - 1]
+    assert state.slot > 0
```

Comment:
Nice

I think we might still need `assert state.slot > PHASE_1_GENESIS_SLOT` because shard transitions from the prior slot do not yet exist. (or at least something handling this strange case)